### DISCUSSION TITLE
/aborting-fetch-requests-spfx-extensions-navigation-events/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#357] [Comments]
Aborting your fetch requests in your SharePoint Framework extensions during navigation events
        
[https://www.eliostruyf.com/aborting-fetch-requests-spfx-extensions-navigation-events/](https://www.eliostruyf.com/aborting-fetch-requests-spfx-extensions-navigation-events/)

**Comment by NewGHUser4321** [comment]
Thanks for the post. It is a good way to abort all the fetch requests whose signal is set. Do you know how we can selectively abort the fetch requests. Lets say I have a search bar and after I type next character, I have to cancel all my previous fetch requests for search except the latest one. I know its been a while a since you posted this. Any help is appreciated.

Thank you.

**Reply by NewGHUser4321** [reply]
Never mind. I got it worked.

**Reply by estruyf** [reply]
Great to hear you got it working. Anything that can be better explained in the article?

**Reply by NewGHUser4321** [reply]
Everything is good. BTW shoutout to you for writing these articles. Three of your articles helped me in my SPFx app development.

**Reply by estruyf** [reply]
Thank you for the feedback!
